Makefile

```diff
@@ -127,21 +127,22 @@ BUILDER_IMAGE := velero-builder
 
 # Example: make shell CMD="date > datefile"
```

Comment:
We want to have a make target to verify that the `go.[mod sum]` don't go out of date.

Reply:
https://github.com/vmware-tanzu/velero/pull/2144/files#diff-b67911656ef5d18c4ae36cb6741b7965R295

Reply:
:+1: I'll borrow from the CSI plugin.